FirebaseRemoteConfigSwift/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.0.0
+- [added] Added a new dynamic property wrapper API that enables developers to create a property wrapper property that listens to a remote config key value and automatically updates the UI component when new config are activated. (#10155)
```

Comment:
nit: maybe this can be a bit shorter? e.g: "Added a new dynamic property wrapper API that enables developers to configure UI components to automatically update when remote config is activated"

Reply:
Done.